app/views/groups/index.html.erb

```diff
@@ -1,5 +1,24 @@
 <h1>All groups</h1>
 
+<script type="text/javascript">
+  $(document).ready(function(){
+    $('#mySel').change(function(){
```

Comment:
Try to avoid using id's whenever possible! Consider that this is the neighborhood dropdown so consider a class name like `'neighborhoodDropdown'`. Be descriptive 😉